packages/react-badge/src/Badge/useBadgeStyles.ts

```diff
@@ -0,0 +1,103 @@
+import { makeStyles, ax } from '@fluentui/react-make-styles';
+import { BadgeState } from './Badge.types';
+
+/**
+ * Styles for the root slot
+ */
+export const useRootStyles = makeStyles<BadgeState>([
+  [
+    null,
+    theme => ({
+      display: 'inline-flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      color: theme.global.color.white,
+    }),
+  ],
+  [
+    s => s.size === 'smallest',
+    theme => ({
+      width: '6px',
+      height: '6px',
+      fontSize: '4px',
+    }),
+  ],
+  [
+    s => s.size === 'smaller',
+    theme => ({
+      width: '10px',
+      height: '10px',
+      fontSize: '6px',
+    }),
+  ],
+  [
+    s => s.size === 'small',
+    theme => ({
+      width: '16px',
+      height: '16px',
+      fontSize: '10px',
+    }),
+  ],
+  [
+    s => s.size === 'medium',
+    theme => ({
+      height: '20px',
+      alignItems: 'center',
+      justifyContent: 'center',
+      gap: '4px',
+      border: '1px',
+      paddingRight: '8px',
+      paddingLeft: '8px',
+    }),
+  ],
+  [
+    s => s.size === 'large',
+    theme => ({
+      width: '24px',
+      height: '24px',
+      fontSize: '12px',
+    }),
+  ],
+  [
+    s => s.size === 'larger',
+    theme => ({
+      width: '32px',
+      height: '32px',
+      fontSize: '12px',
+    }),
+  ],
+  [
+    s => s.size === 'largest',
+    theme => ({
+      width: '32px',
+      height: '32px',
+      fontSize: '12px',
+    }),
+  ],
+  [s => s.circular, theme => ({ borderRadius: theme.global.borderRadius.circular })],
+  [s => s.rounded, theme => ({ borderRadius: theme.global.borderRadius.medium })],
+  [s => s.status === 'success', theme => ({ backgroundColor: theme.global.palette.green.primary })],
+  [s => s.status === 'accent', theme => ({ backgroundColor: theme.global.palette.blue.primary })],
+  [s => s.status === 'danger', theme => ({ backgroundColor: theme.global.palette.cranberry.primary })],
```

Comment:
IMO we should use `.alias` colors there, otherwise theme switching will not work...